openapi/shopify.admin/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to [Shopify Admin API](https://shopify.dev/api/admin-rest) from Ballerina
+
+## Package overview
+The `ballerinax/shopify.admin` is a [Ballerina](https://ballerina.io/) connector for Shopify Admin API.
+This package provides the capability to access Shopify Admin API.
+
+### Compatibility
+|                               | Version                         |
+|-------------------------------|---------------------------------|
+| Ballerina Language            | Ballerina Swan Lake Beta3       | 
+| Shopify Admin API             | 2021-10                         |
+
+## Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina connector repository](https://github.com/ballerina-platform/ballerinax-openapi-connectors)
```

Comment:
This has to point to extended-library repo

Reply:
Resolved. Thank you.